src/bindings/mod.rs

```diff
@@ -38,7 +38,6 @@ pub type MPI_T_enum = *mut Struct_mca_base_var_enum_t;
 pub type MPI_T_pvar_handle = *mut Struct_mca_base_pvar_handle_t;
 pub type MPI_T_pvar_session = *mut Struct_mca_base_pvar_session_t;
 pub type MPI_Win = *mut Struct_ompi_win_t;
-pub type ompi_mpi_comm_world = Struct_ompi_communicator_t;
```

Comment:
removing this because it is a type and not an object, doesn't allow for a comm world to be created correctly.